js/script.js

```diff
@@ -1,8 +1,16 @@
-$(function () {
-    $(document).scroll(function () {
-        var $nav = $(".fixed-top");
-        $nav.toggleClass('scrolled', $(this).scrollTop() > $nav.height());
-        var $text = $(".text-white");
+$(function() {
+    $(document).scroll(function() {
+        var $nav = $('.fixed-top')
+        $nav.toggleClass('scrolled', $(this).scrollTop() > $nav.height())
+        var $text = $('.text-white')
         $text.toggleClass('dark', $(this).scrollTop() > $nav.height())
```

Comment:
There should be space after the function name, and double quotes and single quotes can be used interchangeably, so please remove these changes.

Reply:
ok!